confluence/server/scio_search/src/main/java/com/askscio/atlassian_plugins/confluence/impl/ScioWebhookTask.java

```diff
@@ -32,9 +45,24 @@ public void run() {
       final String json = String.format("{\"url\":\"%s\",\"user\":\"%s\"}", visitedUrl, user);
       final byte[] bytes = json.getBytes(StandardCharsets.UTF_8);
       output.write(bytes);
-      logger.debug(String.format("Webhook sent %d", connection.getResponseCode()));
+      int responseCode = connection.getResponseCode();
+      updatePluginStatus(responseCode);
```

Comment:
do we need a try-catch for this -- just to prevent telemetry from affecting normal functionality ?

Reply:
makes sense, added this safety net